backend/core/src/listings/types/listing-filter-keys-enum.ts

```diff
@@ -10,6 +10,7 @@ export enum ListingFilterKeys {
   maxRent = "maxRent",
   ami = "ami",
   leasingAgents = "leasingAgents",
+  include_nulls = "include_nulls",
```

Comment:
Nit: It would be nice if this was camel-case to match the others.

Reply:
I tried changing the case but it kept error-ing out as it built the URL with the camelcase and not the include_nulls.